packages/0xcert-ethereum-gateway-contracts/src/tests/actions-gateway/misc/cancel.test.ts

```diff
@@ -110,46 +113,28 @@ spec.beforeEach(async (ctx) => {
   const jane = ctx.get('jane');
   const bob = ctx.get('bob');
   const cat = ctx.get('cat');
+  const id1 = ctx.get('id1');
 
   const actions = [
```

Comment:
Please add cancel tests. 

* Valid order
  - Attempt cancel at expiration - 1 ( expect cancel succeeds)
  - Attempt cancel at expiration ( expect cancel fails)
  - Attempt cancel at expiration + 1 ( expect cancel fails)

* Invalid order
  - Attempt cancel at expiration - 1 ( expect cancel fails)
  - Attempt cancel at expiration ( expect cancel fails)
  - Attempt cancel at expiration + 1 ( expect cancel fails)

Reply:
When canceling an order there is only one limitation. That the order has not yet been performed. Otherwise you can cancel the same order multiple times as explained in the smart contract:
``` 
   * @notice You can cancel the same order multiple times. There is no check for whether the order
   * was already canceled due to gas optimization. You should either check orderCancelled variable
   * or listen to Cancel event if you want to check if an order is already canceled.
```

Reply:
@fulldecent good?

Reply:
Got it. My bad. Comment resolved.